environments/dev-env/README.md

```diff
@@ -0,0 +1,395 @@
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/README.md)
+
+
+## Quickstart
```

Comment:
It looks to me like this is leading with the all-in-one rather than the 3 part articulated scenario (1: create infrastructure, 2: deploy edge platform, 3: connect infrastructure and prescribe workload and its placement).  I think we should lead with the 3-part because that is less confusing for newbies (it makes the different roles clearer and walks them through how to use the platform).